FabricExample/ios/FabricExample.xcodeproj/project.pbxproj

```diff
@@ -566,7 +566,7 @@
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = i386;
```

Comment:
I believe it was changes because of problems with building the app on M1, but it seems to be working correctly without it. Removed in https://github.com/software-mansion/react-native-gesture-handler/pull/1893/commits/5fee6b1760d9c9f060f45b26149ca335a36fc8f7.